lgn-worker/src/main.rs

```diff
@@ -727,22 +724,6 @@ fn run_with_websocket(config: &Config) -> Result<()>
             },
         )?;
 
-    // Fetch checksum file
-    // The checksum file can be generated in two ways.
-    // 1- Run the worker, and it will download and spit out the checksum on disk
-    // 2- Manually download the params then install with the checksums bin crate and run checksums
-    // -c -r zkmr_params -a BLAKE3
-    let checksum_url = &config
-        .public_params
-        .checksum_url;
-    let expected_checksums_file = &config
-        .public_params
-        .checksum_expected_local_path;
-    fetch_checksum_file(
-        checksum_url,
-        expected_checksums_file,
-    )?;
```

Comment:
I think downloading the file with the checksums is not necessary if the checksum won't be checked.